binaryaudit/cli.py

```diff
@@ -6,7 +6,7 @@
 arg_parser_common = argparse.ArgumentParser(add_help=False)
 arg_parser_common.add_argument('-v', '--verbose', action='store_true',
                                help="Verbose output.")
-
+arg_parser_common.add_argument("--no-suppressions", action="store_true", help="Disable any default suppressions")
```

Comment:
Suggest to name this `--no-default-suppressions` as that's what this exactly does.

Also, referencing the other commend, perhaps a common functionality could be made as a separate argument parser, which would contain both suppression related arguments.

Thanks.